controllers/database/generate.go

```diff
@@ -78,9 +81,8 @@ func (postgres *PostgreSQLReconciler) generatePostgresCR() (*unstructured.Unstru
 }
 
 //generateHarborDatabaseSecret returns database connection secret
-func (postgres *PostgreSQLReconciler) generateHarborDatabaseSecret(conn *Connect, secretName string) *corev1.Secret {
-
-	return &corev1.Secret{
+func (postgres *PostgreSQLReconciler) generateHarborDatabaseSecret(conn *Connect, secretName, propertyName string) *corev1.Secret {
```

Comment:
if I want to generate Clair Secret, can I write the logic in this function?

Reply:
yes